tests/test_quantity.py

```diff
@@ -0,0 +1,146 @@
+from pytest import approx
+
+import ansys.fluent.core.quantity as q
+
```

Comment:
for some non-matching units conversions, you can check for exceptions
https://stackoverflow.com/questions/23337471/how-to-properly-assert-that-an-exception-gets-raised-in-pytest

You could test Hz etc

Reply:
Thank you for sharing. We will try to check all the possible restricted conversions while writing new test cases.

Reply:
Done.